massa-bootstrap/src/lib.rs

```diff
@@ -12,7 +12,6 @@
 #![warn(unused_crate_dependencies)]
 #![feature(ip)]
 #![feature(map_first_last)]
-#![feature(let_chains)]
```

Comment:
They reverted the stabilization of if let chains so if you update your rust version it will not compile. As the CI run with the last nightly we have to let the features

Reply:
Ok, thanks.